### PR TITLE
feat(seo): add title page hook effects

### PR DESCRIPTION
**Pull request by griest024**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

#1704 

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

**Review by damienwebdev — APPROVED** [review]
lgtm

File: libs/seo/state/src/state.module.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import { DaffSeoTitleEffects } from './effects/title.effects';
+
+@NgModule({
+  imports: [
+    EffectsModule.forFeature([
+      DaffSeoTitleEffects,
+    ]),
+  ],
+})
+export class DaffSeoStateModule {}
```
Comment (damienwebdev) [inline libs/seo/state/src/state.module.ts:13]: Does this need to be in the API?

**Review by damienwebdev — CHANGES_REQUESTED** [review]

File: libs/seo/state/src/state.module.ts (hunk 1 above)
Comment (griest024) [reply libs/seo/state/src/state.module.ts:13]: I suppose not, apps can simply load the effects themselves

Comment (griest024) [reply libs/seo/state/src/state.module.ts:13]: should I remove it?

Comment (griest024) [reply libs/seo/state/src/state.module.ts:13]: oh nvm I see what you mean, yeah the export was in a different commit, I'll add it

**Review by damienwebdev — APPROVED** [review]
lgtm

**Pull request merged**